README.md

```diff
@@ -73,145 +72,101 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-- Added the English abbreviations as fallbacks in the UnitConverter plugin. Thanks [@Tantalus13A98B5F](https://github.com/Tantalus13A98B5F)!
+- Support drag and dropping for file results. Thanks [@daniel-richter](https://github.com/daniel-richter)!
 
 ### Quick Accent
 
-- Added mappings for the mu, omicron, upsilon and thorn characters.
-- Added a setting to exclude apps from activating Quick Accent.
-- Fixed an issue causing the selector to trigger when leaving the lock screen. Thanks [@damienleroy](https://github.com/damienleroy)!
-- Added the Croatian, Netherlands, Swedish and Welsh character sets. Thanks [@damienleroy](https://github.com/damienleroy)!
-- Added support for more unicode characters. Thanks [@char-46](https://github.com/char-46)!
-- Shift-space can now navigate backwards in the selector. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
-- Added the Catalan accented characters. Thanks [@ivocarbajo](https://github.com/ivocarbajo) and [@codingneko](https://github.com/codingneko)!
-- Added the Kurdish accented characters.
-- Added the Serbian accented characters. Thanks [@damienleroy](https://github.com/damienleroy)!
-- Added the Irish and Scottish accented characters.
-- Added the description for the currently selected character in the selector.
-- Fixed a bug causing the selector window to appear blank.
-
-### Runner
-
-- Fixed a crash on a racing condition accessing the IPC communication with Settings.
+- Added support for dark theme. Thanks [@niels9001](https://github.com/niels9001)!
+- Increased default input delay to improve out of the box experience.
+- Fixed a bug causing the first character to not be selected when opening the overlay. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Fixed the positioning of the overlay when showing in the diagonal corners.
+- Added additional Pinyin characters. Thanks [@char-46](https://github.com/char-46)!
+- Added Macedonian characters. Thanks [@ad-mca-mk](https://github.com/ad-mca-mk)!
+- Added a setting to sort characters by usage frequency.
+- Added a setting to always start selection in the first character, even if using the arrow keys as the activation method.
 
 ### Settings
 
-- Fixed settings name in the QuickAccent page. Thanks [@htcfreek](https://github.com/htcfreek)!
-- Added a message indicating there's no network available when looking for updates.
-- Fixed an error causing the backup/restore feature to not find the backup file. Thanks [@jefflord](https://github.com/jefflord)!
-- Fixed localization for the "All apps" expression in the keyboard manager page.
-- UI refactoring, clean-up and bringing in modern controls. Thanks [@niels9001](https://github.com/niels9001)!
-- Improved settings/OOBE screens text. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
-- The backup/restore feature also backs up FancyZones layouts.
+- Show the option to keep the display on when using the "Indefinitely Awake" mode.
```

Comment:
```suggestion
- Added the option to keep the display on when using the "Indefinitely Awake" mode.
```

Reply:
Here it's really "show", because it was being hidden and it was an error.

Reply:
Thanks for all the feedback!

Reply:
```suggestion
- Fixed an error that hid the option to keep the display on when using the "Indefinitely Awake" mode.
```

How about this?